docs(pie-docs): update engineer and foundation pages copy (#343)
justeattakeaway/pie: apps/pie-docs/src/engineers/guidelines/browser-support.md

## Browser support

> Users have access to hundreds of browsers, so we've come up with a rating system to measure the level of browser support we want to achieve and the level of testing needed.

Reviewer: I'd argue we're not really _measuring_ the level of support, because it's something we've defined. Also, I think a bit more explanation helps this paragraph say what it's meaning to say.

```suggestion
Users have access to hundreds of browsers, so instead of assessing them all individually we've come up with a rating system which defines different levels of browser support and the testing requirements for each level. We've assigned the highest ratings to the browsers that are most commonly used by our customers.
```

Reviewer 2: Good call, we can make this change. I'll update the desgins. Thanks!

Reviewer 3: 👍

> Users have access to hundreds of browsers, so instead of assessing them all individually we've come up with a rating system which defines different levels of browser support and the testing requirements for each level. We've assigned the highest ratings to the browsers that are most commonly used by our customers.

Reviewer 4: Without trying to be pedantic, `Users have access to hundreds of browsers` isn't really true. They have access to hundreds of versions of browsers, but not actual browser variations. Just doesn't read correctly to me having gone through the copyrighting.

Reviewer 3: I've updated it to 'hundreds of versions of browsers'

{% notification {
type: "warning",

> message: "This list needs to be reviewed often,  so we can stay up-to-date with the browsers our users are choosing."

Reviewer: When used like this in a sentence it's not hyphenated, only if it's directly before a noun, i.e., "an up-to-date list" vs "keeping the list up to date".
```suggestion
message: "This list needs to be reviewed often to stay up to date with the browsers currently chosen by our users."
```

} %}

---

## Priority definitions

### Priority level A = Fully supported
- Testing is required
- All content must be available
- Layout must comply with the design, unless technical needs prevent this
- All functionality must be available and work as required

### Priority level B = Mostly supported
- Testing is required
- User should be able to place an order or carry out the main function of the page
- Layout doesn't have to be identical to the design, but should retain its functionality

### Priority level C = Not supported
- Testing is not required

{% notification {
type: "information",

> message: "Browsers rated at B or C may have a higher user experience than expected."

Reviewer 2: Good call and good catch!

> message: "Browsers rated at B or C may have a higher user experience than expected."

Reviewer 3: 👍
